src/mraid.ts

```diff
@@ -482,4 +481,8 @@ export class MRAIDImplementation implements MRAIDApi, SDKApi {
       }
     }
   }
+
+  private isInlineVideoSupported(): boolean {
+    return typeof HTMLVideoElement !== undefined;
```

Comment:
I think `typeof` returns a string, so it should be `"undefined"`

Reply:
Great catch. Fixed it!